# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/gobject-introspection/system-glib.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Looks like it wants to provide its own glib version by default instead
of using the system one which overwrites the existing files.

  - NoTag <notag@maneto.fr>

--- gobject-introspection/meson.build.vanilla	2023-04-01 17:46:14.793922157 +0200
+++ gobject-introspection/meson.build	2023-04-01 17:48:32.347612069 +0200
@@ -272,7 +272,7 @@
   description : 'GObject Introspection',
   subdirs: ['gobject-introspection-1.0'],
   variables : pkgconfig_variables,
-  libraries : [glib_dep, gobject_dep],
+  libraries : [gobject_dep],
 )
 
 # This is legacy for backward compat. It is identical to the pc file generated
@@ -288,7 +288,7 @@
   description : 'GObject Introspection',
   subdirs: ['gobject-introspection-1.0'],
   variables : pkgconfig_variables,
-  libraries : [girepo_lib, glib_dep, gobject_dep],
+  libraries : [girepo_lib, gobject_dep],
 )
 
 # Dependency object used by Meson's GNOME module. This dependency variable must
